src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2024-12-06'
+version: 7.6.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.6.2.zip'
+---
+
+## New Features
+
+- Implemented OpenTelemetry (Otel) Distributed Tracing.
+- Added support for Gzip compression for log data ingestion.
+- Included ANR (Application Not Responding) thread data in the reporting process.
```

Comment:
- Included Application Not Responding (ANR) thread data in the reporting process.

Reply:
looks good